string behavior
int32 behavior_code
geometry_msgs/Point32 target
geometry_msgs/Point32[] targets
int32 tower_target
float32 priority
